File: sdk/java/src/main/java/com/kucoin/universal/sdk/model/UnifiedWsArgs.java
package com.kucoin.universal.sdk.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.HashMap;
import java.util.Map;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@Data
@NoArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class UnifiedWsArgs {
  /**
   * User-defined ID (not orderid) is used to uniquely represent a request. The server will also
   * return this ID when returning.
   */
  @JsonProperty("id")
  private String id;

  /**
   * Command options The \"op\" parameter is one of the following enum values.
   *
   * <ul>
   *   <li>spot.order
   *   <li>margin.order
   *   <li>futures.order
   *   <li>spot.cancel
   *   <li>margin.cancel
   *   <li>futures.cancel
   *   <li>futures.multi_cancel
   *   <li>futures.multi_order
   *   <li>spot.sync_order
   *   <li>spot.modify
   *   <li>spot.sync_cancel
   * </ul>
   *
   * @see <a href="https://www.kucoin.com/docs-new/3470133w0">docs</a>
   */
  @JsonProperty("op")
  private String op;

  /** Business parameters, same as RestAPI */
  @JsonProperty("args")
  private Map<String, Object> args = new HashMap<>();
}

File: sdk/php/src/Model/UnifiedWsArgs.php
<?php

namespace KuCoin\UniversalSDK\Model;

use JMS\Serializer\Annotation\Type;
use JMS\Serializer\Annotation\SerializedName;
use JMS\Serializer\Serializer;

/**
 * Class UnifiedWsArgs
 * Represents the arguments for a unified WebSocket API call.
 */
class UnifiedWsArgs
{
    /**
     * User-defined ID (not orderid) is used to uniquely represent a request.
     * The server will also return this ID when returning.
     * 
     * @var string
     * @Type("string")
     * @SerializedName("id")
     */
    public $id;

    /**
     * Command options. The "op" parameter is one of the following enum values:
     * - spot.order
     * - margin.order
     * - futures.order
     * - spot.cancel
     * - margin.cancel
     * - futures.cancel
     * - futures.multi_cancel
     * - futures.multi_order
     * - spot.sync_order
     * - spot.modify
     * - spot.sync_cancel
     * 
     * @var string
     * @Type("string")
     * @SerializedName("op")
     */
    public $op;

    /**
     * Business parameters, same as RestAPI
     * 
     * @var array
     * @Type("array")
     * @SerializedName("args")
     */
    public $args = [];

    /**
     * @param string $json
     * @param Serializer $serializer
     * @return self
     */
    public static function jsonDeserialize(string $json, Serializer $serializer): UnifiedWsArgs
    {
        return $serializer->deserialize($json, self::class, 'json');
    }

    /**
     * @param Serializer $serializer
     * @return string
     */
    public function jsonSerialize(Serializer $serializer): string
    {
        return $serializer->serialize($this, 'json');
    }
}